krping.c

```diff
@@ -1805,7 +1813,7 @@ static void krping_fr_test(struct krping_cb *cb)
 	inv.send_flags = IB_SEND_SIGNALED;
 	
 	DEBUG_LOG("fr_test: stag index 0x%x plen %u size %u depth %u\n", mr->rkey >> 8, plen, cb->size, cb->txdepth);
-	start = get_seconds();
+	start = ktime_get_seconds();
```

Comment:
since you call ktime_get_seconds() here, can you now remove get_seconds()?

Reply:
Actually, I added a new function named `get_seconds()` which calls `ktime_get_seconds()`, just trying to minimize the changes to your original code.